### PR TITLE
[Event] Add links to Hacktoberfest events page

### PR DESCRIPTION
**Pull request by Chadha93**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit ae152852090e4f7809b48a192933a0f767626a67 at: https://6515941053e12712b626cb12--layer5.netlify.app

File: src/collections/events/2023/hacktoberfest-2023/index.mdx [hunk 1]
```diff
@@ -30,6 +30,14 @@ Prepare for Hacktoberfest and ready yourself for contributing to CNCF projects t
 
 <p> Resources:
     <ul>
-    <li><Link to="https://docs.google.com/spreadsheets/d/11KPLU9vVklgGn9NbluLegbfQAyBA5ZS58AhoA8_PCVI/edit#gid=0">Issues for Hacktoberfest</Link></li>
+    <li>
+    <Link to="https://github.com/issues?page=1&q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Ahacktoberfest+user%3Ameshery+user%3Alayer5io+user%3Aservice-mesh-performance+user%3Aservice-mesh-patterns">Hacktoberfest labeled issues: </Link>
```
Comment (Chadha93) [inline src/collections/events/2023/hacktoberfest-2023/index.mdx:34]:
```suggestion
    <Link to="https://github.com/issues?page=1&q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Ahacktoberfest+user%3Ameshery+user%3Alayer5io+user%3Aservice-mesh-performance+user%3Aservice-mesh-patterns">Hacktoberfest labeled issues</Link>
```

File: src/collections/events/2023/hacktoberfest-2023/index.mdx [hunk 2]
```diff
@@ -30,6 +30,14 @@ Prepare for Hacktoberfest and ready yourself for contributing to CNCF projects t
...
+    </li>
+     <li>
+    <Link to="https://github.com/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22+user%3Alayer5io+user%3Ameshery+user%3Aservice-mesh-patterns+user%3Aservice-mesh-performance">Good first labeled issues: </Link>
```
Comment (Chadha93) [inline src/collections/events/2023/hacktoberfest-2023/index.mdx:37]:
```suggestion
    <Link to="https://github.com/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22+user%3Alayer5io+user%3Ameshery+user%3Aservice-mesh-patterns+user%3Aservice-mesh-performance">Good first labeled issues </Link>
```

File: src/collections/events/2023/hacktoberfest-2023/index.mdx [hunk 3]
```diff
@@ -30,6 +30,14 @@ Prepare for Hacktoberfest and ready yourself for contributing to CNCF projects t
...
+    </li>
+     <li>
+    <Link to="https://github.com/issues?page=3&q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22help+wanted%22+user%3Aservice-mesh-performance+user%3Alayer5io+user%3Ameshery+user%3Aservice-mesh-patterns">Help wanted labeled issues: </Link>
```
Comment (Chadha93) [inline src/collections/events/2023/hacktoberfest-2023/index.mdx:40]:
```suggestion
    <Link to="https://github.com/issues?page=3&q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22help+wanted%22+user%3Aservice-mesh-performance+user%3Alayer5io+user%3Ameshery+user%3Aservice-mesh-patterns">Help wanted labeled issues </Link>
```

**Comment by leecalcote** [comment]
I previously noted that I'll make these updates. I'm closing so as to avoid merge conflict.

**Pull request closed without merging**